src/components/Layout/ContentHeader.vue

```diff
@@ -0,0 +1,78 @@
+<script lang="ts" setup>
+import Icon from '/@/components/UI/Icon.vue'
+
+type HeaderText = {
+  title: string
+  url?: string
+}
+
+interface Props {
+  iconName: string
+  headerTexts: Array<HeaderText>
+  detail: string
+}
+const props = defineProps<Props>()
+</script>
+
+<template>
+  <div :class="$style.container">
+    <div :class="$style.titleContainer">
+      <icon :class="$style.icon" :name="props.iconName" :size="48" />
+      <template v-for="(headerText, index) in props.headerTexts" :key="index">
+        <icon
+          v-if="index !== 0"
+          :class="$style.chevron"
+          name="akar-icons:chevron-right"
+          :size="30"
+        />
+        <router-link :to="headerText.url ?? ''" :class="$style.link">
+          <h1 :class="$style.title">{{ headerText.title }}</h1>
+        </router-link>
+      </template>
+    </div>
+    <p :class="$style.detail">{{ props.detail }}</p>
+  </div>
+</template>
+
+<style lang="scss" module>
+.container {
+  margin: 2rem 0;
+}
+
+.titleContainer {
+  display: flex;
+  align-items: center;
+  color: $color-primary;
+  margin-bottom: 0.5rem;
+}
+
+.title {
+  font-size: 1.5rem;
+}
+
+.link {
+  text-decoration: none;
+
+  &:visited {
+    color: $color-primary;
+  }
```

Comment:
visitedじゃないときにリンクの色が微妙な感じになってたので常にcolor-primaryでいい気がしました
```suggestion
.link {
  text-decoration: none;
  color: $color-primary;
```